### ISSUE TITLE
Spurious "solution has been modified externally" messages when performing certain actions

### ISSUE THREAD

**Issue by isaacwaldron** [icsharpcode/SharpDevelop#677]
I am using SharpDevelop 4.4.2.9479 from the main downloads site on Windows 7, and after I perform certain kinds of actions in a solution the GUI pops up a window saying that the solution has been modified externally with an option to reload or ignore the change.  This occurs after every new file or folder creation or file or folder rename operation.  I am using SharpDevelop for IronPython development so I am unsure if the issue is specific to Python solutions or is more general.

**Comment by giuliohome** [comment]
No, it's not specific to Python solutions: same thing here for C# Developments. I too have always to click the button to keep the old solution open and ignore the change

**Comment by giuliohome** [comment]
At code level, it should be the
```c#
FileChangeWatcher.AskForReload();
```
